src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -43,6 +44,9 @@ public class CertificateManager {
     private final Clock clock;
     private final Map<GetCertificateRequest, CertificateGenerator> certSubscriptions = new ConcurrentHashMap<>();
     private CertificatesConfig certificatesConfig;
+    @SuppressWarnings("PMD.UnusedPrivateField")
+    private CAConfiguration caConfiguration;
```

Comment:
future PR: read/write access to this needs to be synchronized or else you may be reading from multiple configuration instances.